### PR TITLE
Add badges

### PR DESCRIPTION
**Pull request by happz**

### CODE REVIEW

**Comment by happz** [comment]
+1

**Review by mkluson — APPROVED** [review]
LGTM

**Review by mkluson — APPROVED** [review]
LGTM

**Comment by mkluson** [comment]
+1

**Review by mkluson — APPROVED** [review]
+1

**Review by mkluson — CHANGES_REQUESTED** [review]
foo

**Comment by mkluson** [comment]
+1 ok, accepted

**Comment by mkluson** [comment]
+1

**Review by mkluson — COMMENTED** [review]
+1

**Review by mkluson — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -3,6 +3,15 @@ gluetool - A tool for gluing together one-file python modules in a sequential co
 
 ``gluetool`` is a command line centric generic framework useable for glueing modules into pipeline
 
+
+.. image:: https://travis-ci.org/gluetool/gluetool-test.svg?branch=master
+    :target: https://travis-ci.org/gluetool/gluetool-test
+
+.. image:: https://requires.io/github/gluetool/gluetool-test/requirements.svg?branch=master
```
Comment (mkluson) [inline README.rst:10]: THis sucks

File: README.rst [hunk 2]
```diff
@@ -3,6 +3,15 @@ gluetool - A tool for gluing together one-file python modules in a sequential co
...
+     :target: https://requires.io/github/gluetool/gluetool-test/requirements/?branch=master
+     :alt: Requirements Status
```
Comment (mkluson) [inline README.rst:12]: This sucks too

**Review by mkluson — CHANGES_REQUESTED** [review]
please

File: README.rst (hunk 1 above)
Comment (happz) [reply README.rst:10]: trhni si

File: README.rst (hunk 2 above)
Comment (happz) [reply README.rst:12]: +1

**Review by happz — COMMENTED** [review]
abc

**Review by mkluson — APPROVED** [review]
Finally approved. Previous comments had testing purpose.

**Comment by thrix** [comment]
Awesome change!

**Review by thrix — DISMISSED** [review]
Awesome change, really!

**Comment by happz** [comment]
@mkluson @thrix Please, approve once again - I rebased the branch, which in turn invalidates your review :)

**Review by mkluson — APPROVED** [review]
LGTM

**Review by thrix — APPROVED** [review]
LGTM now

**Pull request merged**